src/components/rectificationForm/RectificationForm.tsx

```diff
@@ -143,12 +148,12 @@ const RectificationForm = () => {
           )}
         </div>
         <hr />
-        <p>
+        <p className="rectification-link-to-profile">
           <IconCheckCircle aria-hidden="true" />
           {t('common:fetched-from-profile')}
           <Link
             href={window._env_.REACT_APP_PROFILE_UI_URL}
-            size="M"
+            size="S"
```

Comment:
I would like the text and and link to be same size. Maybe the link could be `size=M` and text `font-size: var(--fontsize-body-m)`?